terraform/workspace_variables/dev.tfvars.json

```
@@ -0,0 +1,8 @@
+{
+  "environment_name": "dev",
+  "key_vault_name": "s165d01-kv",
+  "resource_group_name": "s165d01-rg",
+  "apply_qts_app_name": "apply-qts-dev",
```

```suggestion
  "apply_qts_app_name": "apply-for-qts-in-england-dev",
```

The other apps will be:

```bash
apply-for-qts-in-england-production
apply-for-qts-in-england-preprod
apply-for-qts-in-england-test
```

This follows the same convention from Find a lost TRN: https://github.com/DFE-Digital/find-a-lost-trn#live-environments